### ISSUE TITLE
Color contrast is not accessible enough

### ISSUE THREAD

**Issue by DejazmachMolla** [Georjane/LEO-Workshop-Website#2]
Background color and text on the background are of very close contrast. That will not create a great experience for color-blind users.
